plugins/dropdown/src/dropdownPropsExtractor.ts

```diff
@@ -0,0 +1,28 @@
+import Mustache from 'mustache'
+
+const getMassagedProps = (props: any): any => {
+  const { url, mapper } = props.field.options || ({} as any)
```

Comment:
this needs to be `props.field.options.toJSON()` I believe, is it working without that?

Reply:
it's a `Map` type

Reply:
I had an error stating the `options` didn't have a `toJSON()` method, I could add it back, however I'm not confident it would work

Reply:
was that when running in the app or in tests? e.g. we may just need to make sure the tests use a Map instead of an object when passing to the editor like should be the behavior at runtime

Reply:
That was when running the app, not the tests

Reply:
cool, I'll pull this down and run it in the app and try it out and see if i can replicate